com.github.Xenoveritas.abuse.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.github.Xenoveritas.abuse
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --persist=.abuse
+command: abuse
+rename-desktop-file: abuse.desktop
+rename-appdata-file: abuse.appdata.xml
+rename-icon: abuse
+
+modules:
+  - name: abuse
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DASSETDIR=/app/extra/data
+    sources:
+      - type: git
+        url: https://github.com/Xenoveritas/abuse.git
+        tag: v0.9.0
+        commit: 47dee77120cdcf374d80c1968bdc317d850c2f77
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+      - type: extra-data
```

Comment:
given the licensing you shouldn't need to make this "extra-data".

Reply:
Not everything was released as public domain.

Reply:
doesn't need to bw public domain to be redistributable.

Reply:
I removed it. The repository includes enough to make this playable.

Reply:
I was not asking it be removed. I was just asking if having it as extra-data was necessary instead of directly in the package. extra-data is by design less reliable, and is only meant for non-redistributabe elements.